app/views/mailers/pipeline_mailer/error_email.html.erb

```diff
@@ -0,0 +1,98 @@
+<table
+  style="width:100%;border-collapse:collapse;border:0;border-spacing:0;background-color:#f8fafc;"
+>
+  <tbody>
+    <tr>
+      <td align="center" style="padding-top:48px;padding-bottom:48px;">
+        <a href="<%= root_url %>" style="text-decoration:none;">
+          <span style="font-size:40px;color:#000;font-weight:600;">
+            <%= t(:"general.name") %>
+          </span>
+        </a>
+      </td>
+    </tr>
+    <tr>
+      <td align="center">
+        <table
+          style="width:602px;border-collapse:collapse;border:0;border-spacing:0;text-align:left;background:#ffffff;"
+        >
+          <tbody>
+            <tr>
+              <td style="padding:24px;">
+                <h1 style="font-size:24px;color:#000;font-weight:600;">
+                  <%= t(:".greeting", name: @workflow_execution.submitter.first_name) %>&#44;
+                </h1>
+              </td>
+            </tr>
+            <tr>
+              <td style="padding:12px 24px;">
+                <p style="color:#334155;line-height:24px;">
+                  <%= t(:".body.start") %>
```

Comment:
Same thing applies as the above comment

Reply:
ca992a0e2e94a9c4212b50bbf93f00b1be7b8ef1